### DISCUSSION TITLE
Data Transfer assigning attributes

### DISCUSSION THREAD

**Discussion by nubbinse** [bagisto/bagisto#9877] [Q&A]
I am not sure if this is the right area, but my question seems to be simple, but I am having a hard time finding the answer.

I have 2000+ skus I would like to import using data transfer, each one has a handful of attributes. Going back in and assigning each one their attributes would be tedious. In the sample csv download it does not address assigning attributes. It accepts the headers for the attributes like "color" but if I put "Red" as the value it will not show up in the product after it is created. I'm sure this has to do with the fact that it has a dropdown or that it's a "Select" instead of a "Text" field.

Any help would be greatly appreciated

Thank you

**Comment by nubbinse** [comment]
So in case anyone ever wanted to know, we figured out that if you use the ID from the  "attribute_options" table in the database it will work for the data transfer. So initially in the demo store "Red" is 1 and "XL" is 9. so if you create headers for "color" and "size" you put in the values of "1" and "9" your product would have those selections after being imported.

**Comment by amit-webkul** [comment]
Hello, @nubbinse 

I note your concern and am very grateful to you for your question regarding Bagisto.

When importing your 2000+ products, you can easily assign attributes (such as color and size) as needed, not just RED (ID: 1) and XL (ID: 9). You can also add other colors and sizes that you have created to your attributes. You need to refer to the `attribute_options` table, where you can see the created attribute IDs. You can easily set a specific color or size for particular products by using these IDs.

After importing the products, you can see that the attributes are already assigned to the products.

If you have any questions regarding this, feel free to ask here. We will reply as soon as possible.

**Comment by amit-webkul** [comment]
Based on the information provided in the previous response, it appears that the solution has been addressed. Consequently, I will consider this matter resolved and proceed to close the discussion for now.
